tests/autoyast/clone.pm

```diff
@@ -24,12 +24,10 @@ sub run {
     my $self = shift;
     assert_script_run 'rm -f /root/autoinst.xml';
     script_run("(yast2 --ncurses clone_system; echo yast2-clone_system-status-\$?) | tee /dev/$serialdev", 0);
-    assert_screen(['yast2_console-finished', 'autoyast2-install-accept'], 400);
-    if (match_has_tag('autoyast2-install-accept')) {
-        assert_screen 'autoyast2-install-accept';
+    if (check_screen 'autoyast2-install-accept', 10) {
```

Comment:
`check_screen` with non-zero timeout can again introduce timing dependant behaviour as well as is costing test additional test execution time as well. I would have preferred the old way. Let's see if this makes a problem on our slower test environments.